frontend/src/hooks/useTimezones.ts

```diff
@@ -0,0 +1,26 @@
+import TimezoneCountries from 'countries-and-timezones';
+import { useEffect, useState } from 'react';
+import { ISelectOption } from '../component/common/GeneralSelect/GeneralSelect';
+
+export interface ITimezoneSelect extends ISelectOption {
+    utcOffset: string;
+}
+export const useTimezones = (): ITimezoneSelect[] => {
```

Comment:
Why is this a hook without props, that is setting things on `useEffect`? Can this be just a getter function? Why does this require internal state?

Reply:
Getter makes sense. Thanks